Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -3344,7 +3349,7 @@ def gen2021HiMix(fragment,howMuch):
 steps['HARVESTPROMPTRUN3']=merge([{'--data':'', '-s':'HARVESTING:@allForPrompt','--era':'Run3'},steps['HARVESTDRUN3']])
 
 
-steps['HARVESTRUN3']=merge([{'--data':'', '-s':'HARVESTING:@miniAODDQM+@nanoAODDQM'},steps['HARVESTDRUN3']])
+steps['HARVESTRUN3']=merge([{'--data':'', '-s':'HARVESTING:@standardDQM+@miniAODDQM+@nanoAODDQM','--era':'Run3'},steps['HARVESTDRUN3']])
```

Comment:
Are you sure on `HARVESTING:@standardDQM` ?
Does it not look on `TriggerResults::HLT` which is not what you expect, i.e. relvals workflow run with reHLT, i.e. https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/relval_standard.py#L452

Reply:
Thanks Phat! Then which DQM sequence will be an appropriate choice?

Reply:
Hi @sunilUIET 

Sorry, I don't know. The above is only my guess from skimming quickly on configuration. I may be wrong.
You may clarify with @cms-sw/dqm-l2 @cms-sw/hlt-l2